app/models/rdv.rb

```diff
@@ -74,6 +77,21 @@ def send_reminder
     end
   end
 
+  def send_web_hook
+    organisation.webhook_endpoints.each do |w|
+      WebhookJob.perform_later(to_detailed, w)
+    end
+  end
+
+  def send_web_hook_on_destroy
+    rdv = to_detailed
```

Comment:
`rdv` variable name is pretty confusing, I'd expect it to contain an actual Rdv instance. maybe `rdv_serialized`? anyhow, I think this code can be removed cf below !